lib/cylc/suite_logging.py

```diff
@@ -20,19 +20,27 @@
 """
 from __future__ import print_function
 
+import os
+import sys
 import glob
 import logging
 import logging.handlers
-import os
-import sys
 from time import time
-
-
-from cylc.cfgspec.glbl_cfg import glbl_cfg
 import cylc.flags
+from cylc.cfgspec.glbl_cfg import glbl_cfg
 from cylc.wallclock import (get_time_string_from_unix_time,
                             get_current_time_string)
 
+# Suite log filenames.
+SUITE_LOG = 'log'
+SUITE_OUT = 'out'
+SUITE_ERR = 'err'
```

Comment:
Note: The running suite should no longer write anything to `out` these days. I still want to revisit #2505 and sort out suite logging once and for all.

Reply:
Yes.  I don't think SUITE_OUT is actually used anywhere, but the associated logger is defined if we do ever have a reason to use it.  Maybe that's its just superfluous next to SUITE_LOG, but best to leave for your revisit of 2505?

Reply:
Yes. I would like to unify ERR and LOG to just LOG in the code eventually. We can still have an err file if necessary, but that should be configured via handler and filter, etc.